CHANGELOG.md

```diff
@@ -8,32 +8,78 @@ core section.***
 - ***When possible please provide the number of the pull request(s) containing the 
 changes in the following format: PR #1234***
 
-### LORIS 23.0 (Released: ??)
+## LORIS 23.0 (Release Date: ??)
 
+### Core
+#### Features
+- Add configuration option to specify the format in which dates are displayed. (PR #5004)
+- Configurable upload directory for data_release and document_repository modules (PR #5815)
+- New radio components for forms. (PR #5846)
 
-#### Core
+#### Updates and Improvements
+- PHP minimum requirements raised to PHP 7.3 (PR #5723)
 - Menus are now maintained by modules and no longer in the SQL database (PR #5839)
-- Very old instruments relying on QuickForm may have issues due to code changes (PR #4928)
-- Unix user permissions have been updated which may affect access to files. New
-documentation for file permissions has been added to the README.md file (PR #5323)
+- New script created for streamlining instantiation of the Raisinbread demonstration 
+database (PR #5260)
+- New documentation for file permissions has been added to the README.md file. (PR #5323)
```

Comment:
I think the phrasing for this was better before. The use of permissions have changed, not just our documentation for them.

Reply:
could you make a suggestion for this ?

Reply:
@johnsaigle

Reply:
@johnsaigle I looked at the PR and I only see a install and readme change. both of which dont really impact any project already running making this part `Unix user permissions have been updated which may affect access to files` confusing to me